src/gpl/src/replace.tcl

```diff
@@ -321,6 +321,25 @@ proc global_placement { args } {
 }
 
 
+sta::define_cmd_args "cluster_flops" {\
+    [-tray_weight tray_weight]\
+    [-timing_weight timing_weight]\
+    [-max_split_size max_split_size]\
+}
+
+proc cluster_flops { args } {
+  sta::parse_key_args "cluster_flops" args \
+    keys { -tray_weight -timing_weight -max_split_size }
+
+  if { [info exists keys(-tray_weight)] } {
+    set tray_weight $keys(-tray_weight)
+    set timing_weight $keys(-timing_weight)
+    set max_split_size $keys(-max_split_size)
+    gpl::replace_run_mbff_cmd $max_split_size $tray_weight $timing_weight
+  }
```

Comment:
It appears you are treating all the flags as mandatory - otherwise you get a silent no-op.  Is this intentional?

Reply:
Yes -- all these flags must exists or else flip-flop clustering shouldn't run. The user must define alpha, beta and split-size.

Reply:
Are there no reasonable defaults?

Reply:
fixed in latest commit (default: tray_weight = 20.0, timing_weight = 1.0, max_split_size = -1)